Source/Xamarin/Prism.Forms/Behaviors/MultiPageNavigationBehavior.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Prism.Logging;
+using Xamarin.Forms;
+
+namespace Prism.Behaviors
+{
+    /// <summary>
+    /// Provides a MultiPageNavigationBehavior when dealing with any MultiPage&lt;Page&gt; such as a TabbedPage
+    /// </summary>
+    public class MultiPageNavigationBehavior : MultiPageNavigationBehavior<Page>
```

Comment:
I had left this named as MultiPage as it works with any MultiPage<Page>... however I can update this to reflect the naming with the ActiveAware behaviors so this would become ```TabbedPageNavigationBehavior```